busted/modules/files/terra.lua

```diff
@@ -1,7 +1,7 @@
 local path = require 'pl.path'
 
 local ret = {}
-local ok, terralib = pcall(function() return require 'terralib' end)
+local ok, terralib = not not terralib, terralib --pcall(function() return require 'terralib' end)
```

Comment:
I think at least the commented out code should be removed.

Reply:
Done.